pyvista/plotting/plotting.py

```diff
@@ -1846,7 +1837,7 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
             return actors
 
         if not isinstance(volume, pyvista.UniformGrid):
-            raise TypeError('Type ({}) not supported for volume rendering at this time. Use `pyvista.UniformGrid`.')
+            raise TypeError('Type {} not supported for volume rendering at this time. Use `pyvista.UniformGrid`.'.format(type(volume)))
```

Comment:
Good catch.  Perhaps change to:
```python
raise TypeError('Type "{}" not supported for volume rendering at this time. Use `pyvista.UniformGrid`.'.format(type(volume)))
```

Reply:
The output of type looks like `<class '...'>`, so additional stuff around it seems superfluous to me.

Reply:
Good point.